### DISCUSSION TITLE
Demo Days en Español Discussion

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [community/community#34719] [Other Feature Feedback, Questions, & Ideas]
Queremos desarrolladores más productivos, pero también más felices, únete para cuantificar cómo GitHub impacta en la experiencia de los equipos y en conjunto a un ambiente más ágil y eficiente.

Evento en vivo: https://www.linkedin.com/video/event/urn:li:ugcPost:6978774969702838274/

![sep30](https://user-images.githubusercontent.com/20666190/193294961-d1e67d33-1a99-42b1-9083-d3b8cff5ff82.png)

**Comment by oscartobarrios** [comment]
Hola, gracias por el espacio, es muy constructivo. Una Pregunta. La revision de seguridad en el codigo para que lenguajes esta disponible?

**Comment by jveraduran** [comment]
Hola! 
Yo solo quería entregarles mi feedback. Es maravilloso ver como gente con la que compartes, con la que trabajas y co-creas tiene estos espacios para compartir con la comunidad, sus conocimientos y experiencias, y más aun en tu lenguaje nativo. Instancias como estas hacen que estemos mas cerca de ustedes y del producto, por lo cual espero sigan con esa cadencia, y sigamos generando espacios para compartir y colaborar. ❤️ 

Gracias nuevamente a **Lorena** y **Michael** por su tiempo y dedicación, y si hay algo que no puedo hacer, Michael `can too` help me! ⭐

**Reply by AndreaGriffiths11** [reply]
Muchas gracias por compartir @jveraduran ❤️

**Comment by AndreaGriffiths11** [comment]
Hola @oscartobarrios, CodeQL admite lenguajes compilados e interpretados, y puede encontrar vulnerabilidades y errores en el código que está escrito en los lenguajes compatibles.

- C/C++
- C#
- Go
- Java
- JavaScript/TypeScript
- Python
- Ruby
